src/modules/balance/ModuleBalanceCard.vue

```diff
@@ -308,7 +308,7 @@ export default {
      * returns @String
      */
     verifyAddressBody() {
-      return `To verify, check the address on your ${this.instance.meta.name} device & make sure it is the same address as the one shown below.`;
+      return `To verify, check the address on your ${this.instance?.meta.name} device & make sure it is the same address as the one shown below.`;
```

Comment:
move the check to a computed property and return an empty string if instance is null.